features/support/document_helper.rb

```diff
@@ -299,7 +299,9 @@ def content_store_has_business_readiness_finder
 
   def content_store_has_business_readiness_email_signup
     finder_fixture = File.read(Rails.root.join('features', 'fixtures', 'business_readiness_email_signup.json'))
-
+    finder_fixture_without_facets = JSON.parse(finder_fixture)
```

Comment:
I'm getting slightly confused here as to why the feature test is failing.  If the frontend changes haven't been made, then there should be no facets to select.  In which case, we can leave the tests as they are, then modify to allow selection of facets when the frontend change is made.  Or have I completely misunderstood this commit?

Reply:
We added filters in `features/fixtures/business_readiness_email_signup.json` so it'll try and display those. They weren't finalised filters (and isn't the point of this PR) so I'm working on another PR to properly include those. Arguably, yes, we could choose to not include the facets here, inject the facets into the relevant tests and in my PR to include the facets, remove those injections.

Reply:
We can either:

a) leave as-is. Once we've added facets to the frontend, remove this hack & the "no facets" assertion. The fixture data is likely to change once the facets are finalised, but not by much.
b) undo our changes to the fixture data and instead hack the facets and `combine_mode` into the fixture data for our tests only (this is likely to have a negative impact on the readability of the tests we've added). Once we've added facets to the frontend, remove the "no facets" assertion, update the fixture data with the finalised facets and finally remove our hacking of extra facets.

Option b feels marginally more correct but is a fair bit of extra work and for no tangible benefit IMO. Let me know what you think 👍